BibBuddy/app/src/main/java/de/bibbuddy/HomeFragment.java

```diff
@@ -1,38 +1,151 @@
 package de.bibbuddy;
 
+import android.content.Context;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.TextView;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.RecyclerView;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
 
 /**
- * The HomeFragment is responsible for TODO.
+ * The HomeFragment is responsible for welcoming the user and displaying recently used books.
  *
- * @author Claudia Schönherr
+ * @author Claudia Schönherr, Luis Moßburger
  */
-public class HomeFragment extends Fragment {
+public class HomeFragment extends BackStackFragment
+    implements BookRecyclerViewAdapter.BookListener {
 
   private View view;
+  private Context context;
+
+  private BookDao bookDao;
```

Comment:
```suggestion
```
Dao-classes should not be used in our Fragments and Activities (see further comments in this review).

Reply:
Thanks, done!

Reply:
Thanks, done!